flink/v1.15/flink/src/test/java/org/apache/iceberg/flink/TestFlinkCatalogTable.java

```diff
@@ -277,12 +275,11 @@ public void testDowngradeTableToFormatV1ThroughTablePropertyFails() throws Excep
     Table table = table("tl");
     TableOperations ops = ((BaseTable) table).operations();
     Assert.assertEquals("should create table using format v2", 2, ops.refresh().formatVersion());
-
-    AssertHelpers.assertThrowsRootCause(
-        "should fail to downgrade to v1",
-        IllegalArgumentException.class,
-        "Cannot downgrade v2 table to v1",
-        () -> sql("ALTER TABLE tl SET('format-version'='1')"));
+    Assertions.assertThatThrownBy(() -> sql("ALTER TABLE tl SET('format-version'='1')"))
+        .as("should fail to downgrade to v1")
```

Comment:
I think in most cases we don't need `.as()` because the check is self-explanatory

Reply:
Changes made as suggested

Reply:
Changes made as suggested